.github/workflows/release.yml

```diff
@@ -127,22 +127,28 @@ jobs:
 
       - name: Get PR Description
         id: get_pr_description
```

Comment:
One suggestion for the author is to consider adding more error handling in case the API call to `github.pulls.get` fails. It's always good to anticipate and handle potential errors to make the code more robust.